### PR TITLE
hotfix: cursor query

### PR DESCRIPTION
**Pull request by xsteadybcgo**
## Context

* the cursor can be `undefined`, should take it into account

## Were any new external packages added?

* List external packages added and why they were added
No

### CODE REVIEW

File: packages/app/src/subql/Subquery.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class Query {
     orderBy: string = 'BLOCK_HEIGHT_DESC',
     cursor?: string
   ): string {
-    const after = cursor ? `after: "${cursor}"` : ''
+    const after = cursor ? `after: "${decodeURIComponent(cursor)}"` : ''
```
[the remark this answers is not in the document]
Comment (xsteadybcgo) [reply packages/app/src/subql/Subquery.ts:28]: To follow the specifications

**Comment by BruceChar** [comment]
I cannot get this pr for and why this pr be pushed. Can we have more context about the decodeURIComponent() ?

**Comment by xsteadybcgo** [comment]
> I cannot get this pr for and why this pr be pushed. Can we have more context about the decodeURIComponent() ?

@BruceChar  The front-end should encode the parameters by `encodeURIComponent`, we need to decode this. [Referrence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)

**Comment by xsteadybcgo** [comment]
I deleted it. The data here doesn't seem to need to be encoded.

**Review by BruceChar — APPROVED** [review]

**Pull request merged**